### DISCUSSION TITLE
Update Rollup Recipe

### DISCUSSION THREAD

**Discussion by waynevanson** [gulpjs/gulp#2584] [Ideas]
Hi All,

It's [recommended by rollup](https://github.com/rollup/stream#usage-with-gulp) to use `@rollup/stream` instead of `rollup-stream`.

The [following recipe](https://github.com/gulpjs/gulp/blob/master/docs/recipes/rollup-with-rollup-stream.md) is outdated. Solution is to advise users to redirect themselves to [`@rollup/stream` readme.md](https://github.com/rollup/stream#usage-with-gulp) or write a recipe similar to what they've written.

I've spent 4 days writing a gulp stream for rollup because I didn't know rollup had their own stream implementation, unlucky.

**Comment by rootwork** [comment]
@waynevanson I think this is more than an idea, this is a documentation bug. I know documentation in general is in flux and being reviewed (#2164), but maybe worth opening a PR for this quick fix?

**Comment by phated** [comment]
@waynevanson PR #2503 was attempting to do this but the author had other stuff come up and didn't finish it. Happy to review any adjustments.
